### PR TITLE
Update the custom_model's README.md after recent logging changes (#2884)

### PR DESCRIPTION
**Pull request by elukey**
"Add description of the problem that #2884 solves"

## Proposed Changes 

- Tried to clarify the ModelServer's logging behavior of Kserve after #2884.

### CODE REVIEW

File: docs/modelserving/v1beta1/custom/custom_model/README.md [hunk 1]
```diff
@@ -111,9 +111,11 @@ You can supply additional command arguments on the container spec to configure t
...
+- `--enable_latency_logging`: Whether to log latency metrics per request, the default is True.
+- `--configure_logging`: Whether to configure KServe and Uvicorn logging, the default is True.
+In this case you may want to set the KServe `ModelServer`'s `log_config` parameter to pass a dictionary containing all the logging directives and configurations (see [the Python upstream docs](https://docs.python.org/3/library/logging.config.html#dictionary-schema-details) for more info). The alternative is to use the `--log_config_file` argument described below.
+- `--log_config_file`: The path of the Python config file configuration to use (either a json or a yaml file). This file allows to override the default Uvicorn configuration shipped with KServe. The default is None.
```
Comment (yuzisun) [inline docs/modelserving/v1beta1/custom/custom_model/README.md:117]: Oh we should rename the argument 

https://github.com/kserve/kserve/blob/f9e9a079712e202e11855419a03b2966320fe575/python/kserve/kserve/model_server.py#L95

File: docs/modelserving/v1beta1/custom/custom_model/README.md [hunk 2]
```diff
@@ -111,9 +111,11 @@ You can supply additional command arguments on the container spec to configure t
...
+- `--access_log_format`: A string representing the access log format configuration to use. The functionality is provided by the `asgi-logger` library and it allows to override only the `uvicorn.access`'s format configuration with a richer set of fields (output hardcoded to `stdout`). This limitation is currently due to the ASGI specs that don't describe how access logging should be implemented in detail (please refer to this Uvicorn [github issue](https://github.com/encode/uvicorn/issues/527) for more info). By default is None.
 
 #### Environment Variables
```
Comment (yuzisun) [inline docs/modelserving/v1beta1/custom/custom_model/README.md:120]: We also added an environment variable KSERVE_LOG_LEVEL to set log level

File: docs/modelserving/v1beta1/custom/custom_model/README.md (hunk 1 above)
Comment (elukey) [reply docs/modelserving/v1beta1/custom/custom_model/README.md:117]: I thought the same at first, but then I realized that we will not probably be able to pass a dictionary in here right? So stating the log_config_file may be more precise. As alternative we could renamed it to `--log_config` and state that it is related only to a file path, is this what you mean?

File: docs/modelserving/v1beta1/custom/custom_model/README.md (hunk 2 above)
Comment (elukey) [reply docs/modelserving/v1beta1/custom/custom_model/README.md:120]: Worth to mention, will check it!

File: docs/modelserving/v1beta1/custom/custom_model/README.md (hunk 1 above)
Comment (yuzisun) [reply docs/modelserving/v1beta1/custom/custom_model/README.md:117]: I think we should rename the argument, seems like you can pass the arg as dict 
https://stackoverflow.com/questions/18608812/accepting-a-dictionary-as-an-argument-with-argparse-and-python

File: docs/modelserving/v1beta1/custom/custom_model/README.md (hunk 2 above)
Comment (elukey) [reply docs/modelserving/v1beta1/custom/custom_model/README.md:120]: Added! Lemme know if I have to add more or not.

File: docs/modelserving/v1beta1/custom/custom_model/README.md (hunk 1 above)
Comment (elukey) [reply docs/modelserving/v1beta1/custom/custom_model/README.md:117]: Created https://github.com/kserve/kserve/pull/2927, let's discuss what we want to do in there and then I'll update this PR.

Comment (yuzisun) [reply docs/modelserving/v1beta1/custom/custom_model/README.md:117]: As discussed in the PR we are going to leave as is as specifying a large json dict on yaml might not be good idea

**Comment by yuzisun** [comment]
Thanks @elukey for this doc update !

/lgtm
/approve

**Pull request merged**